bin/functions/build.sh

```diff
@@ -51,16 +51,18 @@ EOH
 
 function build_uberjar_proj {
     PROJ=$1
-    echo "Building '$PROJ' uberjar ..."
+    cprintln $RED2$GREEN$UNDERLINE "Building '$PROJ' uberjar ..."
+    
     if [ "$CMR_INTERNAL_NEXUS_REPO" = "true" ]; then
+        cprintln $RED$BLINK "with nexus"
         (cd $CMR_DIR/$PROJ && \
             lein with-profile +uberjar,+internal-repos do clean, uberjar)
     else
-        (cd $CMR_DIR/$PROJ && \
+        (cd $CMR_DIR/$PROJ && pwd && \
             lein with-profile +uberjar do clean, uberjar)
     fi
     if [ $? -ne 0 ] ; then
-        echo "Failed to generate '$PROJ' uberjar" >&2
+        cprintln $RED "Failed to generate '$PROJ' uberjar" >&2
```

Comment:
is this my code?

Reply:
That is your code, it was there when I took over the branch